index.html

```diff
@@ -282,6 +201,44 @@ <h2>Contact me</h2>
+    <section id="popup-section">
+      <div class="popup-container popup-section" id="popup">
+        <div class="popup-top">
+          <h2 class="project-name" id="popup-name">Popup</h2>
+          <a class="close-popup" href="#">
+            <svg width="24" height="24" fill="currentColor" stroke="currentColor" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M5.293 5.293a1 1 0 011.414 0L12 10.586l5.293-5.293a1 1 0 111.414 1.414L13.414 12l5.293 5.293a1 1 0 01-1.414 1.414L12 13.414l-5.293 5.293a1 1 0 01-1.414-1.414L10.586 12 5.293 6.707a1 1 0 010-1.414z" fill="#fff" /></svg>
+          </a>
+        </div>
+        <div class="work-header-items">
+          <p class="work-header-item-one work-header-item-margin">CANOPY</p>
+          <span class="work-header-item-round work-header-item-margin"></span>
+          <p class="work-header-item-margin">Back End Dev</p>
+          <span class="work-header-item-round work-header-item-margin"></span>
+          <p class="work-header-item-margin">2015</p>
+        </div>
+        <img src="./assets/img/work/SnapshootPortfolio1.png" alt="tonic" class="popup-image" id="popup-image" />
+        <div class="popup-content">
+          <p id="popup-content"></p>
+          <div class="popup-elements">
+            <div class="popup-tags" id="popup-tags"></div>
+            <hr />
+            <div class="popup-buttons">
+              <a class="bg-white-button popup-button" id="see-live">
+                See Live<svg width="18" height="18" fill="none" class="popup-button-svg" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M2 9a7 7 0 017-7 1 1 0 100-2 9 9 0 109 9 1 1 0 10-2 0A7 7 0 112 9zm11-9a1 1 0 100 2h1.586L8.293 8.293a1 1 0 001.414 1.414L16 3.414V5a1 1 0 102 0V1a1 1 0 00-1-1h-4z" fill="#6070FF" /></svg>
+              </a>
+              <a class="bg-white-button popup-button" id="see-source">
+                See Source<svg width="18" height="20" fill="none" class="popup-button-svg" xmlns="http://www.w3.org/2000/svg">
+                  <g clip-path="url(#clip0)"><path d="M17.999 7.461c0-1.14-.366-2.21-1.065-3.143.266-.898.333-2.038.266-3.35C17.167.414 16.735 0 16.202 0c-.3 0-2.828.035-4.292 1.382a10.536 10.536 0 00-3.893 0C6.587.035 4.058 0 3.725 0c-.532 0-.965.415-.998.967-.1 1.313 0 2.453.266 3.35-.698.968-1.064 2.039-1.064 3.144 0 2.418 1.796 4.594 4.525 5.63-.1.208-.167.45-.233.691-2.895-.31-4.26-3.04-4.326-3.143a.942.942 0 00-1.33-.449c-.5.242-.7.864-.433 1.382.067.172 1.896 3.868 5.922 4.283v3.109c0 .587.433 1.036.998 1.036h5.823c.566 0 .998-.45.998-1.036v-4.11c0-.623-.133-1.21-.366-1.728C16.202 12.056 18 9.914 18 7.461z" fill="#6070FF" /></g>
+                  <defs>
+                    <clipPath id="clip0"><path fill="#fff" d="M0 0h18v20H0z" /></clipPath>
+                  </defs>
+                </svg>
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript. Check an exampe of students projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96.
- [ ] Modal popup data is populated with the data from the projects array.

Reply:
Hi @descholar-ceo
Thank you for your time and the detailed review. I would like to point out that I used javascript for the project section HTML and CSS creating. The above section is the modal popup section and its data is populated with the data from the projects elements array via javascript. I didn't delete the default text in the model popup section because I think if the browser doesn't render javascript it will be shown. Sorry for that.